docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -32,22 +32,26 @@ The _gateway nodes_ selection depends on the following criteria:
 - *version*: Remote nodes must be compatible with the cluster they are
 registered to. This is subject to the same rules as <<rolling-upgrades>>.
 Any node can communicate with any other node on the same major version (e.g.
-6.0 can talk to any 6.x node). Only nodes on the last minor version of a
-certain major version can communicate with nodes on the following major
-version (e.g. 6.7 can communicate with 7.0, as well as any 7.x node, while
- 6.6 or earlier cannot talk to any 7.x node). Note that version compatibility
+6.0 can talk to any 6.x node). Only nodes on the last minor version (6.8 is a
+bit special in this context as it was released after 7.0) of a certain major
+version can communicate with nodes on the following major version: 6.7 can
+communicate with 7.0, as well as any 7.x node, while 6.6 or earlier cannot
+talk to any 7.x node; note that 7.1, which was released after 7.0, cannot
+talk to 7.0 but rather to 7.1 and later. Note that version compatibility
  is symmetric, meaning that if 6.7 can communicate with 7.0, 7.0 can also
  communicate with 6.7. The matrix below summarizes compatibility as described
  above.
 
 [cols="^,^,^,^,^,^"]
 |====
-| Compatibility | 5.0->5.5 | 5.6 | 6.0->6.6 | 6.7 | 7.x
-| 5.0->5.5      |    Yes   | Yes |    No    | No  | No
-| 5.6           |    Yes   | Yes |    Yes   | Yes | No
-| 6.0->6.6      |    No    | Yes |    Yes   | Yes | No
-| 6.7           |    No    | Yes |    Yes   | Yes | Yes
-| 7.x           |    No    | No  |    No    | Yes | Yes
+| Compatibility | 5.0->5.5 | 5.6 | 6.0->6.6 | 6.7 | 6.8 | 7.0 | 7.1->7.x
+| 5.0->5.5      |    Yes   | Yes |    No    | No  | No  | No  |    No
+| 5.6           |    Yes   | Yes |    Yes   | Yes | Yes | No  |    No
+| 6.0->6.6      |    No    | Yes |    Yes   | Yes | Yes | No  |    No
+| 6.7           |    No    | Yes |    Yes   | Yes | Yes | Yes |    Yes
```

Comment:
I think the 7.1->7.x column should be No here? We guarantee the last minor fo the previous major. Now that 6.8 is out, the compatible version checks in 7.1+ would see 6.8 as the minimum compatible version, not 6.7.

Reply:
good to know, I was not sure.